common/cron/cron_test.go

```diff
@@ -29,59 +29,59 @@ var _ = Describe("Cron", func() {
 		It("Test every second cron", func() {
 			job := Every(1).Seconds()
 			job.Do(DummyTask)
-			Expect(job.String()).To(Equal("[JOB]type: 1(seconds), interval: 1, delay: 0, at: , task: servicecenter/common/cron_test.DummyTask"))
+			Expect(job.String()).To(ContainSubstring("[JOB]type: 1(seconds), interval: 1, delay: 0, at: ,"))
```

Comment:
This testcase specifically checks the job name and job name is decided by the path from which the code is running, the path can change for the forked branches so to make it generic this change was made.